### PR TITLE
templates/*: small a11y improvements to more semantic html and adding…

### PR DESCRIPTION
**Pull request by phillimorland**
… labels to custom filter and adding skiplink

header.scss: don't display moving logo on smaller screens

### CODE REVIEW

**Review by goapunk — APPROVED** [review]
Haven't tested with screenreader but looks good to me! Will leave for a second reviewer

File: apps/academy/templates/academy/academy_index_page.html [hunk 1]
```diff
@@ -10,17 +10,17 @@
...
                 <div class="academy-list__intro">
                     {{ page.translated_intro | richtext }}
                 </div>
             </div>
-        </div>
+        </header>
 
         <div class="row justify-content-center">
-
+          <h2 class="visually-hidden">Filter list by topic</h2>
```
Comment (khamui) [inline apps/academy/templates/academy/academy_index_page.html:23]: indentation (too less)?

File: apps/academy/templates/academy/academy_index_page.html [hunk 2]
```diff
@@ -86,8 +86,8 @@ <h1>{{ page.translated_title }}</h1>
     <div class="container">
         <div class="row filter-line academy-list__sorting">
             <div class="col-sm-6 col-lg-3 mb-4 px-0 px-sm-3">
-                <div class="filter-line__prefix academy__sans-serif ms-1">{% trans 'Type of content:' %}</div>
-                <button class="filter-line__current filter-line__current--small">{% firstof get_academy_content_type_display _('all') %}</button>
+                <label class="filter-line__prefix academy__sans-serif ms-1" for="filter-id">{% trans 'Type of content:' %}<span class=visually-hidden>filter</span></label>
```
Comment (khamui) [inline apps/academy/templates/academy/academy_index_page.html:89]: `visually-hidden` not wrapped in `"..."`

Comment (khamui) [reply apps/academy/templates/academy/academy_index_page.html:89]: still works though

File: apps/academy/templates/academy/academy_index_page.html [hunk 3]
```diff
@@ -10,17 +10,17 @@
 <section class="academy-list">
```
Comment (khamui) [inline apps/academy/templates/academy/academy_index_page.html:10]: should we add aria-label?

File: apps/core/templates/core/home_page.html [hunk 4]
```diff
@@ -24,7 +24,7 @@
 
 {% blog_entries %}
 
-    <div class="row-right align-items-center">
+    <section class="row-right align-items-center">
```
Comment (khamui) [inline apps/core/templates/core/home_page.html:27]: aria-label here as well?

**Review by khamui — COMMENTED** [review]
cool! just some comments

File: apps/academy/templates/academy/academy_index_page.html (hunk 2 above)
Comment (phillimorland) [reply apps/academy/templates/academy/academy_index_page.html:89]: weird!

File: apps/academy/templates/academy/academy_index_page.html (hunk 3 above)
Comment (phillimorland) [reply apps/academy/templates/academy/academy_index_page.html:10]: here I don't think so as there are headings to explain

File: apps/core/templates/core/home_page.html (hunk 4 above)
Comment (phillimorland) [reply apps/core/templates/core/home_page.html:27]: here probably yes however if address settings have not been added then it shouldn't have it, i'll check prod and figure out what makes sense

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!

**Pull request merged**